content/download.md

```diff
@@ -26,64 +26,67 @@ limitations under the License.
 
 ## {{ site.data.project.name }} Downloads
 
-There are no official Apache releases for {{ site.data.project.short_name }} yet.
-
-First Apache Incubator release coming soon!!
+The Official Apache release for {{ site.data.project.short_name }}.
```

Comment:
I would just delete this line ("The Official Apache release for Rya") as the website is already for Rya downloads)